### DISCUSSION TITLE
How Liberator sign is made?

### DISCUSSION THREAD

**Discussion by DerpyHoowes** [SmartlyDressedGames/Unturned-3.x-Community#4074] [Q&A]
Its seemingly a Decal, yet in fact its Medium type object.
Interactability doesn't work on decals, so question is: How do I make my own object, that behaves like decal, so I can put Interactability features on it, as I need this for a custom NPC quest?

**Comment by SDGNelson** [comment]
Sorry to say the Liberator decal is manually set up with the Decal component in Unity. There is not yet a good way for mods to create something similar.
